cfgov/v1/image_formats.py

```diff
@@ -0,0 +1,11 @@
+from wagtail.images.formats import Format, register_image_format
+
+
+register_image_format(
+    Format(
+        'bleed',
+        'Bleed into left/right margins',
+        'richtext-image image-bleed',
+        'original'
```

Comment:
Let's set this to the max width of a bleeding image so that it doesn't request originally-uploaded images that are unnecessarily large.

```suggestion
        'width-1170'
```

Reply:
Makes sense 🎉